namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-prod/resources/hmpps-allocations.tf

```diff
@@ -0,0 +1,130 @@
+resource "kubernetes_secret" "hmpps-allocations" {
+  metadata {
+    name      = "hmpps-domain-events-topic-hmpps-allocations"
+    namespace = var.namespace
+    # Remove when namespace has been migrated
```

Comment:
I've had to temporarily comment out all domain references until the referenced namespaces have all been migrated.  Also renamed to ensure they don't conflict in the same namespace.